rules/windows/powershell/powershell_icmp_exfiltration.yml

```diff
@@ -0,0 +1,26 @@
+title: PowerShell ICMP Exfiltration
+id: 4c4af3cd-2115-479c-8193-6b8bfce9001c
+status: experimental
+description: Detects Exfiltration Over Alternative Protocol - ICMP. Adversaries may steal data by exfiltrating it over an un-encrypted network protocol other than that of the existing command and control channel.
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1048.003/T1048.003.md#atomic-test-2---exfiltration-over-alternative-protocol---icmp
+    - https://attack.mitre.org/techniques/T1048/003/
```

Comment:
Hello @bczyz1! Good to see you here (:
The rule looks perfect, I would only ask you to remove the link to the ATT&CK techniques from the references, as it is already linked in the `tags`.
Thank you for your contribution!

Reply:
Hey @yugoslavskiy, thanks, it's good to be here ;)
I've removed the redundant ref in b6876e51235f913521235ade5e9a036bb98d7074.